### PR TITLE
ba-100 improve recipe styling

### PR DESCRIPTION
**Pull request by patrick-hebner**
@flashback2k14  @lyio

### CODE REVIEW

File: app/elements/socobo-recipe/socobo-recipe-details-show.html [hunk 1]
```diff
@@ -78,19 +79,36 @@
     }
 
     /* TITLE */
+    #title_default {
```
Comment (flashback2k14) [inline app/elements/socobo-recipe/socobo-recipe-details-show.html]: Please only use classes for styling.

Comment (patrick-hebner) [inline app/elements/socobo-recipe/socobo-recipe-details-show.html]: Why, if there is only one occurrence? But ok, doesn‘t bother me

Comment (flashback2k14) [inline app/elements/socobo-recipe/socobo-recipe-details-show.html]: - more consistency
- fast find if The element has a styling
- classes for styling
- ids for Dom access

Comment (patrick-hebner) [inline app/elements/socobo-recipe/socobo-recipe-details-show.html]: Ok

File: app/elements/socobo-recipe/socobo-recipe-details-show.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
       min-width:700px;
       padding: 0;
       margin-top : 20px;
-      margin-bottom: 0;
+      margin-bottom: 20px;
       overflow: scroll;
```
Comment (flashback2k14) [inline app/elements/socobo-recipe/socobo-recipe-details-show.html]: please change overflow to auto. with scroll the scrollbars always visible.

**Comment by flashback2k14** [comment]
looks nice, but the new buttons are really hard to read. 

new-button

**Comment by flashback2k14** [comment]
could you please add mixins for all your colors!? this makes it a lot easier to change colors in one global file (app-theme.html).

syntax: var(--ELEMENT-NAME-Descripton-Name, Default Value);

And add the Mixins and Descriptions in the Docu Tag.

**Comment by patrick-hebner** [comment]
Maybe that's better

**Comment by flashback2k14** [comment]
fab mini and rounded images corners? 

new-button2

**Comment by patrick-hebner** [comment]
I add the mixins. But only for the whole recipe component. If I would add some mixins for each element I would repeat myself to much.

**Comment by patrick-hebner** [comment]
So I have created the mixins and also the related docu entries. I hope thats enough for this pull request

**Comment by lyio** [comment]
Sorry to open this up again, but is everyone on board with the typography?

**Comment by patrick-hebner** [comment]
This was just a little experiment because I found the default font a little bit boring and inappropriate for our domain. I'm open for other ideas and suggestions and I also think we should make a global decision about that issue.

Should I open a typography issue to collect our ideas? 

I would not like to change this again since we are still able to change the font later again.

**Comment by flashback2k14** [comment]
Could you add a mixin for The font? For fast changing? :-)

**Comment by flashback2k14** [comment]
One question. Why you add ba-103 for The issue number? This is all ba-100.

**Comment by patrick-hebner** [comment]
Oh shit I must have copied the number from another braches commit message :(
I can't change this anymore, right?

**Comment by lyio** [comment]
And :+1: for creating a typography issue

**Comment by flashback2k14** [comment]
i.O.

**Pull request merged**